walletunlocker/service.go

```diff
@@ -129,8 +130,7 @@ func (u *UnlockerService) GenSeed(ctx context.Context,
 	// Now that we have our set of entropy, we'll create a new cipher seed
 	// instance.
 	//
-	// TODO(roasbeef): should use current keychain version here
-	cipherSeed, err := aezeed.New(0, &entropy, time.Now())
+	cipherSeed, err := aezeed.New(keychain.KeyDerivationVersion, &entropy, time.Now())
```

Comment:
This line is slightly over 80 chars, our general approach for wrapping something like this would be:
```
	cipherSeed, err := aezeed.New(
                keychain.KeyDerivationVersion, &entropy, time.Now(),
        )
```